examples/Feeds/RemoveFeedItems.cs

```diff
@@ -0,0 +1,111 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.V4.Errors;
+using Google.Ads.GoogleAds.V4.Services;
+
+namespace Google.Ads.GoogleAds.Examples.V4
+{
+    /// <summary>
+    /// Removes feed items from a feed.
+    /// </summary>
+    public class RemoveFeedItems : ExampleBase
+    {
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            RemoveFeedItems codeExample = new RemoveFeedItems();
+            Console.WriteLine(codeExample.Description);
+
+            // The Google Ads customer ID for which the call is made.
+            long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+
+            // The Feed ID to which the feed items belong.
+            long feedId = long.Parse("INSERT_FEED_ID_HERE");
+
+            // The IDs of the feed items to remove.
+            long[] feedItemIds =
+            {
+                long.Parse("INSERT_FEED_ITEM_ID_1_HERE"),
+                long.Parse("INSERT_FEED_ITEM_ID_1_HERE")
+            };
+
+            codeExample.Run(new GoogleAdsClient(), customerId, feedId, feedItemIds);
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description => "Removes feed items from a feed.";
```

Comment:
I like this style, perhaps we can upgrade all our examples to this format. Can you open a maintenance bug for it?

Reply:
Done: https://github.com/googleads/google-ads-dotnet/issues/231